src/main/java/com/hartwig/platinum/kubernetes/ProcessRunner.java

```diff
@@ -12,8 +15,10 @@ boolean execute(List<String> arguments) {
         ProcessBuilder processBuilder = new ProcessBuilder(arguments);
         try {
             LOGGER.debug("Starting [{}]", arguments);
-            Process process = processBuilder.start();
+            Process process = processBuilder.inheritIO().start();
+            BufferedReader reader = new BufferedReader(new InputStreamReader(process.getInputStream()));
             process.waitFor();
+            LOGGER.info(reader.lines().collect(Collectors.joining("\n")));
```

Comment:
this will be a little noisy. can we make it debug?

Reply:
Yep well picked.